internal/svcmanager/svcmanager.go

```diff
@@ -20,20 +21,27 @@ import (
 // MinimalTimeout is the minimum timeout required for requests that are meant to be near-instant
 const MinimalTimeout = 500 * time.Millisecond
 
+var errVersionMismatch = locale.NewError("err_ping_version_mismatch")
+
 type Manager struct {
-	ready bool
-	cfg   configurable
+	ready        bool
+	checkVersion bool
```

Comment:
This isn't doing anything, it's always true as per the logic. Should just drop this.